### DISCUSSION TITLE
Can not update router info: Api call error, status:200; code: -2, category: unauthorized

### DISCUSSION THREAD

**Discussion by a-dongde** [vmakeev/huawei_mesh_router#49] [Q&A]
Hello, my router model is TC7102. When using this plugin, it keeps reporting the following error message

Logger: custom_components.huawei_mesh_router.update_coordinator (Huawei Router)
Source: custom_components/huawei_mesh_router/update_coordinator.py:190
Integration: Huawei Mesh Router (documentation, issues)
First occurred: 10:07:54 (66 occurrences)
Last logged: 10:40:26

Can not update router info: Api call error, status:200; code: -2, category: unauthorized

How can I avoid continuing to report this error?

**Comment by vmakeev** [comment]
Hi! It looks similar to the situation with the Q2S router (https://github.com/vmakeev/huawei_mesh_router/issues/34). When trying to access the router's API via a URL that is not supported by the router, the router itself breaks the connection and gives a "temporary ban" on accessing it.

I'm afraid that right now I don't have the opportunity to modify the component, because I can't reproduce this behavior on the router models that I have. I suspect that this problem is typical for routers aimed at domestic consumers in China, but this is not accurate.

However, if I suddenly get the necessary device at my disposal, or come up with another solution to the problem, I will report it here
